dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/vitals/JankStatsActivityLifecycleListener.kt

```diff
@@ -0,0 +1,82 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.rum.internal.vitals
+
+import android.app.Activity
+import android.app.Application.ActivityLifecycleCallbacks
+import android.os.Bundle
+import androidx.metrics.performance.FrameData
+import androidx.metrics.performance.JankStats
+import com.datadog.android.v2.api.InternalLogger
+import java.util.concurrent.TimeUnit
+
+/**
+ * Utility class listening to frame rate information.
+ */
+internal class JankStatsActivityLifecycleListener(
+    private val vitalObserver: VitalObserver,
+    private val internalLogger: InternalLogger
+) : ActivityLifecycleCallbacks, JankStats.OnFrameListener {
+
+    // region ActivityLifecycleCallbacks
+
+    override fun onActivityCreated(activity: Activity, savedInstanceState: Bundle?) {
+        try {
+            JankStats.createAndTrack(activity.window, this)
```

Comment:
this will work only for activity windows but will not measure the frame rate for `DialogWindow` or `PopUpWindow`. I guess once we add that ticket to support that in RUM we will have to find a different way to plugin in the `JankStats` into those windows too.